swift-blackjack-viewsTests/DeckSpec.swift

```diff
@@ -0,0 +1,48 @@
+//  DeckSpec.swift
+
+import Quick
+import Nimble
+@testable import swift_blackjack_views
+
+class DeckSpec: QuickSpec {
+    override func spec() {
+        describe("Deck") {
+            
+            var deck : Deck = Deck()
+            
+            beforeEach({ () -> () in
+                deck = Deck()
+            })
+            
+            describe("initializer") {
+                it("should set up the deck with 52 undealt cards") {
+                    expect(deck.description).to(contain("Cards Remaining: 52"))
```

Comment:
yeah, it's pretty hacky. i was still finding my way around how Swift handles visibility at the time. I had privatized the arrays, but maybe i can just privatize the setters and make them readonly instead.